README.md

```diff
@@ -1,3 +1,40 @@
 # New York Times Article
 
-First CSS project of Lidya Ghebreigziabher and Abhigyan Mahanta
\ No newline at end of file
+First CSS project of Lidya Ghebreigziabher and Abhigyan Mahanta
+
+<img src="images/projectpage.PNG" alt="NewYork_Times_Article">
+
+Additional description about the project and its features.
```

Comment:
- [ ] If this line is a title, please make it has the same style with the other titles in your README.